docs/configuration/secrets/troubleshoot.md

```diff
@@ -0,0 +1,34 @@
+# Troubleshooting
+
+The sections below include information to help you troubleshoot issues that you may come across while configuring the secrets used by Codacy.
+
+If the information provided on this page is not enough to solve your issue, contact [support@codacy.com](mailto:support@codacy.com) providing all the information that you were able to collect while following the troubleshooting instructions.
+
+## Lost/Changed initially generated secrets
+
+While trying to open Codacy,
+you were shown an error message stating that,
+the secret on the database and the one in your configuration file are different.
+
+### Steps
+
+1. Get the tail of the logs on the api service
```

Comment:
Three comments:

- Why mention tail specifically? I think "Get the latest logs from the api service" would be more appropriate. 
- Not sure if we are using "api" or "API".
- Guarantee that we are always calling 'api service' to the website backend.

Reply:
> Guarantee that we are always calling 'api service' to the website backend

Not sure how to do this.

The only reason I called it api is because the name of the pod is api.